include/sys/spa_impl.h

```diff
@@ -433,6 +433,7 @@ struct spa {
 };
 
 extern char *spa_config_path;
+extern char *zfs_deadman_failmode;
```

Comment:
This can be removed and `zfs_deadman_failmode` can remain private.  It's only accessed from within `spa_misc.c`.

Reply:
```
static int 
zfs_deadman_failmode_sysctl(SYSCTL_HANDLER_ARGS)
{
	char buf[16];
	int rc;

	if (req->newptr == NULL)
		strlcpy(buf, zfs_deadman_failmode, sizeof (buf));

	rc = sysctl_handle_string(oidp, buf, sizeof (buf), req);
	if (rc || req->newptr == NULL)
		return (rc);
	if (strcmp(buf, zfs_deadman_failmode) == 0)
		return (0);
	if (!strcmp(buf,  "wait"))
		zfs_deadman_failmode = "wait";
	if (!strcmp(buf,  "continue"))
		zfs_deadman_failmode = "continue";
	if (!strcmp(buf,  "panic"))
		zfs_deadman_failmode = "panic";

	return (-param_set_deadman_failmode_common(buf));
}
```

Reply:
Ahh, ok.